### ISSUE TITLE
feat(ink): a lazily fetched ink/fx.js with its own 8 KiB budget, the seam every effect hangs on

### ISSUE THREAD

**Issue by agentchieflou** [agentchieflou/this-next-please#370]
**Part of #293.** Effects get a module of their own, `static/ink/fx.js`, fetched only by a skin that has them, so the four modules every drawing desk loads stay inside `INK_BUDGET`.

## Why
The always-fetched ink modules gzip to 40,542 of the 40,960 bytes `INK_BUDGET` allows (ink.js 7,125 + layer.js 18,585 + pen.js 9,980 + shapes.js 4,852; measured at 8557b2b as `test_the_ink_payload_is_inside_its_budget_and_three_is_not_in_it` does: gzip level 6, mtime 0). A cue engine written into ink.js and layer.js measured about 1,022 bytes (41,564, 604 over). The seam below, with the engine in a lazily fetched fx.js, measured +292 bytes (ink.js +73, layer.js +219). The ink budget has one rule, owned by the decisions register #318: `INK_BUDGET` (tests/test_fleet_ink.py:57) is raised once, to 44 KiB, by #331, the first card that needs it, on the operator's register answer (default yes). This card lands after #331 and #345 (ink.js) and fits under 44 KiB; it does not raise it. Every later effects card (#372, #374, #375, #376) writes fx.js only, under `FX_BUDGET` 8 KiB.

## Read first
- `agentdata/fleet/static/ink/ink.js`: `normalise()` 117-213 (its returned object 203-212), `start()` 296-310 (layer.js fetched with `import(q(...))`), `materials()` 313-319, `HOOKS`/`valueOf`/`fromModule` 358-374, `setSkin` 405-410, `window.Ink` 412-433.
- `agentdata/fleet/static/ink/layer.js`: `setTable()` 189-225 (observed attributes at 206), `makeApi()` 263-286 (`order` at 275), `ctx`/`hook` 288-304, `enskin`/`unskin`/`empty` 306-346, `framePanes()` 349-370 (the pane group at 356), `evaluate()` 425-469, `syncAll()` 770-776, `kick`/`now`/`prepare`/`frame` 1141-1218, `refresh()` 1245, `inspect()` 1250-1296 (`skin` is null without a skin, 1287).
- `tests/test_fleet_ink.py`: 41-60 (MODULES, SKINS, SCRIPTS, budgets), 82-90 (`_own_desk_globals`), 136-153 (`_open`; its `asked` list at 146), 204-255 (no-markup, token-import, payload), 395 (`IDLE_LOOP`).
- `docs/desk-ink.md` §The files, §Writing a skin, §Budgets.

## Build
1. **Re-measure first**, at your branch point (the payload test prints the four sizes). If the seam does not fit under `INK_BUDGET` as #331 left it (44 KiB by default), stop and ask: do not trim comments and do not raise the budget.
2. **ink.js.** `normalise()` returns `fx: table.fx || null`. `fromModule()` adds `fx: m.cues || o.fx ? { cues: valueOf(m.cues, variant), use: o.fx } : null` to the table (a skin exports `cues`; `options.fx` is `{text, pointer}` for #375/#376). `HOOKS` gains `"cue"`.
3. **layer.js**, exactly the seam in the sketch: `api.order.fx = -5`; a getter `api.fx`; `setTable` fetches fx.js when `t.fx`; `unskin` detaches it first (so `stop()` and every table change do); `evaluate()` ends with `fx.match()`; `syncAll()` ends with `fx.measure()`; `frame()` calls `fx.deliver()` right after `prepare()`; `inspect()` returns `fx: this.fx && this.fx.inspect()` in its **top-level** object, beside `skin` (not inside `skin`, which is null after `Ink.setSkin(null)`).
4. **New `agentdata/fleet/static/ink/fx.js`**: no static import, never names `vendor/three` (it is handed the layer), no markup. A module-level `let live = 0` and `export function attached() { return live; }`. `export function attach(layer, spec)` makes one `layer.THREE.Group` with `renderOrder = layer.api.order.fx`, adds it to `layer.scene`, increments `live`, and returns `{api, match, measure, deliver, detach, inspect}`: `api` is `Object.freeze({})`; `match`, `measure` and `deliver` do nothing yet; `detach()` empties the group with `layer.empty`, removes it, decrements `live` and returns `null`; `inspect()` returns `{loaded: true, rows: 0, children}`. #372 fills them.
5. **Draw order.** three.js r160 sorts first by the innermost Group's `renderOrder` (`if(t.isGroup)n=t.renderOrder` in the vendored build), and the pane groups `framePanes` makes (layer.js:356) keep renderOrder 0. So the fx group at -5 draws over the back pass (ground, paper) and under every pane's frame and every mark, whatever `api.order.frame` says. Within one list three.js draws every opaque object before any transparent one, so effect materials are `transparent: true` like the skins' own. Write that down; an effect that must sit on a pane's frame goes into that pane's frame group.
6. **tests/test_fleet_ink.py.** `LAZY = ("fx.js",)`, `FX_BUDGET = 8 * 1024`, `SCRIPTS = MODULES + LAZY + skins`. The payload test keeps `INK_BUDGET` for MODULES (as #331 set it), adds `fx.js < FX_BUDGET`, and its listdir assertions become MODULES + LAZY (+ `skins`), with a docstring line saying why.

```js
// prototype from research, not merged: a sketch of the layer.js seam (+292 B gzip at 8557b2b)
order: Object.freeze({ ground: -30, paper: -20, frame: -10, fx: -5 }),
get fx() { return self.fx && self.fx.api; },     // fx.js's helpers (§Effects)
// setTable(), after enskin:
// Effects: their own module, fetched only by a skin that has them (§Budgets).
if (t && t.fx) import(q("/static/ink/fx.js")).then(m => {
  if (this.table === t && !this.stopped) { this.fx = m.attach(this, t.fx); this.refresh(); }
}, e => console.error("ink: fx.js: " + e));
// unskin(), first line:      if (this.fx) this.fx = this.fx.detach();
// evaluate(), before return: if (this.fx) this.fx.match();
// syncAll(), before return:  if (this.fx) this.fx.measure();
// frame(), after prepare():  if (this.fx) this.fx.deliver();
// inspect(), top level:      fx: this.fx && this.fx.inspect(),
```

## Acceptance criteria
- [ ] ink.js + layer.js + pen.js + shapes.js gzip under `INK_BUDGET` (44 KiB after #331; the test prints the total), and `INK_BUDGET` is unchanged by this PR.
- [ ] fx.js gzip under `FX_BUDGET` (8,192).
- [ ] fx.js is requested 0 times for a table without `fx`, exactly once with `?t=<token>` for a table with it, and not again when the same table is set twice (counted in the `asked` list `_open` returns).
- [ ] After a table without `fx`, after `Ink.setSkin(null)`, and after `Ink.off()`: `(await import(q('/static/ink/fx.js'))).attached() === 0` and `!(Ink.inspect().layer && Ink.inspect().layer.fx)`.
- [ ] With fx attached and at rest, `IDLE_LOOP` gives 0 mutations and 0 renders.
- [ ] Every other ink and skin test passes untouched; none is skipped.

## Tests
- tests/test_fleet_ink.py: the budget and listing changes above. `test_the_ink_modules_write_no_markup` and the token-import test now cover fx.js through `SCRIPTS`.
- New tests/test_fleet_ink_fx.py. Import `from test_fleet_ink import (IDLE_LOOP, TABLE, _desk_of, _serve, _stop, _open, fleet_home, _own_desk_globals)  # noqa: F401`, as tests/test_fleet_ink_farmstead.py:35-36 does: the autouse `_own_desk_globals` keeps desk globals from leaking between tests. One test, `@pytest.mark.browser`, `test_fx_js_is_fetched_only_by_a_table_with_effects`: set `TABLE`; then `Ink.setSkin({...TABLE, fx: {cues: []}}, {cue() {}, tick() { return false; }})`, twice; then each detach path in turn (a fresh page for `Ink.off()`). Import fx.js from the test only after the fx table was set, so the import reaches the module the layer already fetched and adds no request. Condition waits only.
- Local: `python -m pytest tests/test_fleet_ink.py tests/test_fleet_ink_fx.py --durations=0` (after `playwright install chromium`).

## Docs
docs/desk-ink.md: §The files (an fx.js row: fetched only when a skin exports `cues` or `options.fx`); §Budgets (an FX_BUDGET row and the measured total); §Writing a skin (`api.order.fx` and what -5 really means, `api.fx`); §Tests.

## Out of scope / do not
- Do not raise `INK_BUDGET`, and do not shorten comments elsewhere to make room. If the seam does not fit, stop and ask.
- No cue logic in ink.js or layer.js beyond the seam; no static import; fx.js never imports three.js.
- No change to how marks, frames or materials behave.

## Depends on
#296, #331, #345

## Decision
The ink budget (#318's one rule): `INK_BUDGET` is raised once, to 44 KiB, by #331 on the operator's register answer (default yes); this seam fits under it; effect code goes into the lazily fetched `ink/fx.js` under `FX_BUDGET = 8 KiB` (built here). No other card raises `INK_BUDGET`. Reversible.
